page/style.css

```diff
@@ -1,223 +1,607 @@
-}
-.bullet-larg-text {
-  font-size: 60px;
-}
-article:nth-child(6) {
-  clear: both;
-  background-color: #524741;
-  padding-top: 6px;
-  font-weight: 100;
-  text-align: center;
-  width: 730px;
-  margin-left: 111px;
-  font-size: 32px;
-  height: 794px;
-}
-article:nth-child(6) > h1 {
-  color: white;
-  font-family: pt sans;
-  font-size: 35px;
-}
-article:nth-child(6) > figure > img {
-  height: 477px;
-  margin-left: -39px;
-}
-article:nth-child(6) > section {
-  font-family: merriweather;
-  font-weight: 300;
-  color: white;
-  font-size: 15px;
-  margin: -19px 23px 21px 23px;
-  text-align: left;
-}
-article:nth-child(6) > aside {
-  text-align: left;
-  margin-left: 23px;
-  font-size: 15px;
-  color: lightgray;
-}
-article:nth-child(7) {
-  margin-left: 120px;
-  position: relative;
-  float: left;
-}
-article:nth-child(7) > figure > img {
-  width: 561px;
-  margin-left: -90px;
-  margin-top: -119px;
-  z-index: -1;
-  position: relative;
-}
-article:nth-child(7) > section {
-  width: 485px;
-  font-family: merriweather;
-  font-weight: 200;
-  font-size: 13px;
-}
-article:nth-child(7) > h1 {
-  font-family: oswald;
-  font-size: 55px;
-  font-weight: 200;
-  border-bottom: 12px solid black;
-  width: 471px;
-  margin-top: -31px;
-}
-article:nth-child(7) > h3 {
-  font-family: oswald;
-  margin-top: -45px;
-  margin-bottom: 14px;
-  font-size: 25.8px;
-  border-bottom: 3px solid black;
-  width: 471px;
-  line-height: 67px;
-}
-article:nth-child(8) {
-
-}
-article:nth-child(8) {
-
-}
-article:nth-child(8) {
-
-}
-article:nth-child(8) {
+	font-family: lato;
+	font-weight: 100;
+	color: white;
+	font-size: 143px;
+	margin-left: 130px;
+	display: inline-block;
+	line-height: .8;
+	z-index: 5;
+	width: 50px;
+	margin-bottom: 158px;
+	margin-top: 10px;
+	position: relative;
+}
+
+.box-model {
+	width: 331px;
+	background-color: #cdc9dd;
+	margin: -445px 0 -28px 23px;
+	height: 637px;
+	padding-bottom: 16px;
+	padding-right: 15px;
+}
+
+.box-model img {
+	width: 276px;
+	margin: 230px 0px -7px 9px;
+	z-index: 6;
+	position: relative;
+}
+
+.box-model h1 {
+	margin-left: 152px;
+	font-family: lora;
+	font-style: italic;
+	font-weight: bold;
+	color: #5f5d66;
+	line-height: 2;
+	margin-top: -15px;
+}
+
+.box-model section {
+	margin-top: -24px;
+	margin-left: 90px;
+	width: 234px;
+	font-family: lato;
+	text-align: center;
+	color: #6e6c77;
+	font-size: 13px;
+	line-height: 1.5;
+	font-weight: 500;
+}
+
+.positioning-relative {
+	position: absolute;
+	top: 63px;
+	left: 271px;
+}
+
+.positioning-relative > h1 {
+	color: #fdcd84;
+}
+
+.positioning-relative > p {
+	color: #fdcd84;
+}
+
+.positioning-relative p {
+	margin-top: -138px;
+	margin-left: 149px;
+	z-index: 10;
+	font-size: 12px;
+	letter-spacing: 2px;
+}
+
+.positioning-relative h1 {
+	font-family: oswald;
+	margin-left: 149px;
+	letter-spacing: 2px;
+	font-size: 35px;
+	width: 30px;
+	margin-top: 7px;
+	line-height: 1.2;
+}
+
+.positioning-relative > a > figure > img {
```

Comment:
Selector depth like this is fragile and memory costly. In this case, how many `img` tags are there? Or it would make more sense to have a selector like `positioning-relative__img`